README.md

```diff
@@ -13,7 +13,7 @@ The following table shows operations which are currently supported by the engine
 | Type                   | Supported                                                                 | Priority |
 |------------------------|---------------------------------------------------------------------------|----------|
 | Binary expressions     | Full support                                                              |          |
-| Histogram quantile     | Full support                                                              |          |
+| Histogram quantile     | Partial support (no support for native histograms)                        | High     |
```

Comment:
Priority High means it is high for native histogram? Or what else are left for Histogram quantile?

Reply:
It means high for native histograms because now we will fail queries on timeseries that have native histograms in their chunks.

Reply:
I see. I don't know if we need to make it high because even Thanos doesn't support the native histogram feature and almost no client has supported instrumenting  native histograms. But I think it is fine to mark high in the doc.